pa/private.go

```
@@ -0,0 +1,420 @@
+	tb.Helper()
+	assert(tb, assertData, cond, "assert")
+	tb.Fail()
+}
+
+func ZZZRequire(tb testing.TB, assertData *assertData, cond bool) {
+	tb.Helper()
+	assert(tb, assertData, cond, "require")
+	tb.FailNow()
+}
+
+const debug = true // TODO: make configurable, default to false
```

golangci-lint

pa/private.go:108: Line contains TODO/BUG/FIXME: "TODO: make configurable, default to fals..."

(godox)